bundles/asdi/asdi-projection-change/component/card.js

```diff
@@ -0,0 +1,65 @@
+Oskari.clazz.define('Oskari.projection.change.component.card',
+/**
+ * @class 
+ * @param { Object } view - contains info about the current view passed down to card
+ * @param function() callback - callback function to call when element is clicked
+ */
+function (view, callback) {
+    this.card = _.template('<div class="projection-card" data-srs="${srs}">'+
+                        '<div class="card-image ${imgCls}"></div> '+
+                        '<div class="info-row">'+
+                            '<p class="card-header"> ${projectionName} </p>'+
+                            '<div class="projection-info icon-info"></div>'+
+                            '<div class="projection-warning" title="${tooltip}"></div>'+
+                        '</div>'+
+                        '</div>');
+    this.element = null;
+    this.loc = Oskari.getLocalization('projection-change');
+    this.callback = callback;
+    this.view = view;
+    this.infoView = Oskari.clazz.create('Oskari.projection.change.view.ProjectionInformation', view);
+    this.errorListing = Oskari.clazz.create('Oskari.projection.change.view.ErrorListing');
+    if ( !this.getElement() )
```

Comment:
This will always be true so you can remove the if.

Reply:
It would make more sense for this if to be at the start of create and just exit the function if element already exists.